test/test_config3.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "program3",
+    "unit": 2,
+    "pre": 3,
+    "cycle": 10,
+    "ratio":{
```

Comment:
`"ratio": {`

Reply:
fixed in https://github.com/sepandhaghighi/nafas/pull/129/commits/48131ab3b26126f16f39c3bf2b5a6a095ca7470b